Styles/Classes/UIElement+Extensions/UITextView+Styles.m

```diff
@@ -127,7 +127,12 @@ - (void)applyTextStyle:(TextStyle *)style {
     if (!text || !style) {
         return;
     }
-    [self setAttributedText:[style applyTo:text]];
+
+	UITextRange *range = self.selectedTextRange;
```

Comment:
align

Reply:
it is -- and all above too -- github shows that as invalid 🤷‍♂️